pkg/apis/crd/v1alpha1/types.go

```
@@ -381,12 +381,22 @@ type UDPHeader struct {
 	DstPort *int32 `json:"dstPort,omitempty"`
 }
 
+// TCPFlagMatcher describes a TCP flags matching filter with flag and mask.
+type TCPFlagMatcher struct {
+	// Flag is the TCP flag value to match.
+	Flag int32 `json:"flag"`
+	// Mask is used to specify which bits to consider.
```

you need to specify what the "default" behavior is (when the mask is omitted)

---

updated